test/Recipe-test.js

```diff
@@ -282,12 +282,19 @@ describe('Recipe', () => {
   it('should return the recipe instructions', () => {
     expect(recipe.getInstructions).to.be.a('function');
     expect(recipe.getInstructions()).to.deep.equal([
-      "1. In a large mixing bowl, whisk together the dry ingredients (flour, pudding mix, soda and salt). Set aside.In a large mixing bowl of a stand mixer, cream butter for 30 seconds. Gradually add granulated sugar and brown sugar and cream until light and fluffy.",
-      "2. Add egg and vanilla and mix until combined.",
-      "3. Add dry ingredients and mix on low just until incorporated. Stir in chocolate chips.Scoop the dough into 1,5 tablespoon size balls and place on a plate or sheet. Cover with saran wrap and chill at least 2 hours or overnight.When ready to bake, preheat oven to 350 degrees.",
-      "4. Place the cookie dough balls into ungreased muffin pan. Sprinkle with sea salt.",
-      "5. Bake for 9 to 10 minutes, or until you see the edges start to brown.",
-      "6. Remove the pan from the oven and let sit for 10 minutes before removing onto a cooling rack.Top with ice cream and a drizzle of chocolate sauce."
+      "In a large mixing bowl, whisk together the dry ingredients (flour, pudding mix, soda and salt). Set aside.In a large mixing bowl of a stand mixer, cream butter for 30 seconds. Gradually add granulated sugar and brown sugar and cream until light and fluffy.",
+      "Add egg and vanilla and mix until combined.",
+      "Add dry ingredients and mix on low just until incorporated. Stir in chocolate chips.Scoop the dough into 1,5 tablespoon size balls and place on a plate or sheet. Cover with saran wrap and chill at least 2 hours or overnight.When ready to bake, preheat oven to 350 degrees.",
+      "Place the cookie dough balls into ungreased muffin pan. Sprinkle with sea salt.",
+      "Bake for 9 to 10 minutes, or until you see the edges start to brown.",
+      "Remove the pan from the oven and let sit for 10 minutes before removing onto a cooling rack.Top with ice cream and a drizzle of chocolate sauce."
```

Comment:
Whoops meant this piece in my previous comment!

Reply:
Good catch on this. Looks good to me!